.github/workflows/export_docs.yml

```diff
@@ -0,0 +1,27 @@
+name: Doc Export
+
+on:
+  push:
+    branches: [ "main" ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  pushdocs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: extract readmes
+        run: node scripts/extract_readmes.js
+      - name: Pushes to another repository
+        uses: cpina/github-action-push-to-another-repository@main
+        env:
+          SSH_DEPLOY_KEY: ${{ secrets.SSH_PUSH_DOC_REPO }}
+        with:
+          source-directory: 'extracted_readmes'
+          destination-github-username: 'CPTNB'
```

Comment:
Historically, we got called out and faced issues about having something that was hosted on a non-OpenSearch Project sanctioned. By including this here it seems like this doc repo is an official repo.

Do you think it might be worth going through the process to creating an official repo or starting a doc directory in this repo and enable github pages and see if we can have custom rules that can be way less strict for that repo.